cbbr_build/tests/test_geocoding.py

```diff
@@ -0,0 +1,184 @@
+# test geocoding logic
```

Comment:
SICK - would be nice to have some comments here as to what the tests are doing and why but this shouldn't hold us up. Maybe a circle back

Reply:
definitely, and I wanna circle back on testing the geocoding in full and not just the "utilities"